packages/client/src/components/LocationPicker.tsx

```diff
@@ -123,7 +122,11 @@ function LocationPickerComponent(props: LocationPickerProps) {
         onClick={() => setModalVisible(true)}
       >
         <ContentWrapper>
-          <span>{selectedSearchedLocation.displayLabel}</span>
+          <span>
+            {selectedSearchedLocation
+              ? selectedSearchedLocation.displayLabel
```

Comment:
@youshamahmood96 this is an expected property, we were not getting it due to some issue on Niue resource package's `/location` api. I have put a fix for the cause here: https://github.com/opencrvs/opencrvs-niue/pull/5